src/main/java/at/ac/tuwien/damap/enums/ELicense.java

```diff
@@ -1,35 +1,50 @@
 package at.ac.tuwien.damap.enums;
 
 public enum ELicense {
-
+    
+    // TODO: link is dead, fix it
+    AGPL1("AGPL-1.0-only", "Affero General Public License v1.0 only", "http://www.affero.org/oagpl.html"),
+    AGPL1PLUS("AGPL-1.0-or-later", "Affero General Public License v1.0 or later", "https://spdx.org/licenses/AGPL-1.0-or-later.html"),
+    AGPL3("AGPL-3.0-only", "GNU Affero General Public License v3.0 only", "https://www.gnu.org/licenses/agpl.txt"),
+    AGPL3PLUS("AGPL-3.0-or-later", "GNU Affero General Public License v3.0 or later", "https://spdx.org/licenses/AGPL-3.0-or-later.html"),
+    APACHE1("Apache-1.0", "Apache License 2.0", "http://www.apache.org/licenses/LICENSE-1.0"),
+    APACHE1_1("Apache-1.1", "Apache License 1.1", "http://apache.org/licenses/LICENSE-1.1"),
+    APACHE2("Apache-2.0", "Apache License 2.0", "http://www.apache.org/licenses/LICENSE-2.0"),
+    ARTISTIC1("Artistic License 1.0", "Artistic License 1.0", "https://opensource.org/licenses/Artistic-1.0"),
+    ARTISTIC1PERL("Artistic-1.0-Perl", "Artistic License 1.0 (Perl)", "http://dev.perl.org/licenses/artistic.html"),
+    ARTISTIC2("Artistic-2.0", "Artistic License 2.0", "http://www.perlfoundation.org/artistic_license_2_0"),
+    BSD2C("BSD-2-Clause", "BSD 2-Clause \"Simplified\" License", "https://opensource.org/licenses/BSD-2-Clause"),
+    BSD3C("BSD-3-Clause", "BSD 3-Clause \"New\" or \"Revised\" License", "https://opensource.org/licenses/BSD-3-Clause"),
+    CCBY("CC-BY-4.0", "Creative Commons Attribution 4.0 International", "https://creativecommons.org/licenses/by/4.0/legalcode"),
+    CCBYNC("CC-BY-NC-4.0", "Creative Commons Attribution Non Commercial 4.0 International", "https://creativecommons.org/licenses/by-nc/4.0/legalcode"),
+    CCBYNCSA("CC-BY-NC-SA-4.0", "Creative Commons Attribution Non Commercial Share Alike 4.0 International", "https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode"),
+    CCBYNCND("CC-BY-NC-ND-4.0", "Creative Commons Attribution Non Commercial No Derivatives 4.0 International", "https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode"),
+    CCBYND("CC-BY-ND-4.0", "Creative Commons Attribution-NoDerivs", "https://creativecommons.org/licenses/by-nd/4.0/legalcode"),
```

Comment:
Shouldnt the name of CCBYND be `Creative Commons Attribution No Derivatives 4.0 International`?

Reply:
Leftover from the previous code...